extensions/amp-sidebar/1.0/component.js

```diff
@@ -166,11 +167,81 @@ function SidebarWithRef(
         >
           <div className={classes.backdropOverscrollBlocker}></div>
         </div>
-      </>
-    )
+      </div>
+    </>
```

Comment:
Previously, the `sidebar` elements are only rendered when the Sidebar is opened.  Updating this to always render the `sidebar` elements, but hiding them in a `display: none` element when Sidebar is closed.

This makes it so that the `SidebarNavToolbar` component which is a child of the `Sidebar` component is always rendered.  This is necessary because this element uses a portal to render content onto the main document body (outside of the `amp-sidebar`).  We do not want this content to conditionally render only when the `Sidebar` is opened.

Reply:
Sounds good, I'll track this as a follow up for now in the tracker: https://github.com/ampproject/amphtml/issues/31366

(In case we change the implementation as a result of further discussion, I think it makes sense to do this separately).